### ISSUE TITLE
Feature request - simultanious downloads

### ISSUE THREAD

**Issue by jamess60** [alexta69/metube#437]
Sometimes I bulk download multiple videos from different sources. Some sites are significantly faster/slower than others. Its annoying to have to wait for the active (slow) download to complete when a fast download could be occurring simultaneously. 

Id like to propose a docker environment variable to enable simultaneous downloads, eg:
`            - SIMULTANEOUS_DOWNLOADS=3 `

**Comment by mng-25** [comment]
Agreed. It would also be nice to be able to make use of the -n flag directly from the UI

**Comment by jamess60** [comment]
bump

**Comment by bilalwaheedch** [comment]
This would be an amazing addition.

**Comment by drd2aiki** [comment]
agreed!

**Comment by jamess60** [comment]
@alexta69 Any news on this?

**Comment by alexta69** [comment]
@jamess60 https://github.com/alexta69/metube?tab=readme-ov-file#submitting-feature-requests

**Comment by alexta69** [comment]
Done in #602
